CMakeLists.txt

```diff
@@ -16,17 +16,20 @@ get_git_date(GIT_COMMIT_DATE)
 set(BUILT_FROM "${GIT_DESCRIBE} - Commit: ${GIT_COMMIT_HASH} - Date: ${GIT_COMMIT_DATE}")
 message(STATUS ${BUILT_FROM})
 
-if ((CMAKE_HOST_UNIX) AND (${CMAKE_HOST_SYSTEM_NAME} MATCHES "CYGWIN*"))
+if (((CMAKE_HOST_UNIX) AND (${CMAKE_HOST_SYSTEM_NAME} MATCHES "CYGWIN*")) OR WIN)
   message(STATUS "Building on CYGWIN CMAKE: Adapting paths")
   set(CYGWIN 1)
   set(WIN 1)
 
   # specify the cross compiler
-  set(CMAKE_TOOLCHAIN_PREFIX x86_64-w64-mingw32)
-
-  SET(CMAKE_C_COMPILER   ${CMAKE_TOOLCHAIN_PREFIX}-clang)
-  SET(CMAKE_CXX_COMPILER ${CMAKE_TOOLCHAIN_PREFIX}-clang++)
-  SET(CMAKE_RC_COMPILER ${CMAKE_TOOLCHAIN_PREFIX}-windres)
+  set(CMAKE_TOOLCHAIN_PREFIX "")
+
+  SET(CMAKE_C_FLAGS "-target x86_64-pc-windows-gnu")
+  SET(CMAKE_REQUIRED_FLAGS "-target x86_64-pc-windows-gnu")
+  SET(CMAKE_CXX_FLAGS "-target x86_64-pc-windows-gnu")
```

Comment:
@Alisu sorry for coming so late here.
Do you remember why this flags are needed?

Reply:
I don't really remember, @tesonep  guided me for this.